src/store/selectors.ts

```diff
@@ -11,7 +11,14 @@ export const getEthBalance = (state: StoreState) => state.blockchain.ethBalance;
 export const getWethBalance = (state: StoreState) => state.blockchain.wethBalance;
 export const getOrders = (state: StoreState) => state.relayer.orders;
 export const getUserOrders = (state: StoreState) => state.relayer.userOrders;
-export const getSelectedToken = (state: StoreState) => state.relayer.selectedToken;
+export const getSelectedToken = (state: StoreState) => {
+    const selectedToken = state.relayer.selectedToken;
```

Comment:
@mariano-aguero I would prefer not to transform the returned value of this function: what happens if we want to check for equality between `getSelectedToken(state).symbol` with a given `token.symbol`? (we will probably have to convert to uppercase the 2nd one every time we do this).

Should we do this on the React component instead? We can use JS `string.toUpperCase()` or CSS `text-transform: uppercase` accordingly.

We can also change the tokens' metadata file and set the symbols in upper case (and `wETH` with its special-casing :confused:). For this 2nd option, probably we should check with @fvictorio and @Agupane since I may be missing something

Reply:
Yes, this should be done in the component, it's totally a view concern.

We could add it to the metadata... but just uppercasing the symbol should be enough for now. We are already handling `wETH` as a special case everywhere, so doing it once more here for the label shouldn't be a big deal.

Reply:
According to what I saw, this could be a good choice,  I made some tests and seems OK. but I'm not 100% sure if this could generate some unexpected behaviour.

Reply:
Plus, this seems to be mutating the store, doesn't it? Shouldn't `readonly` prevent us from doing that, or the `readonly`ness is lost when you copy a value?

Reply:
Nevermind, `readonly` is just applied at the top level. The point remains: we shouldn't mutate the store directly.

Reply:
The store was modified to its previous state, , and a CSS transformation was added on the React component.